docs/content/en/docs/concepts/support-versions.md

```diff
@@ -53,6 +52,38 @@ The following table notes which EKS Anywhere and related Kubernetes versions are
 
 * [Amazon EKS Anywhere Enterprise subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/) is required to receive AWS support on any Amazon EKS Anywhere clusters.
 
+## EKS Anywhere versions and bundles
+
+Each EKS Anywhere version installs a suite of componenets when provisioning your clusters. The versions of these components are specified in an EKS Anywhere versions bundle.
+
+You may find a map of EKS Anywhere releases in the release manifest located [here](https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml). It contains references to the corresponding bundle manifest for each EKS Anywhere version.
```

Comment:
```suggestion
You may find a map of EKS Anywhere releases in the release manifest located in the [manifest.yaml](https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml) file. It contains references to the corresponding bundle manifest for each EKS Anywhere version.
```
In general, I think it's best to avoid "here" and attach a name to the link you are point to.